src/excel/utilities.ts

```diff
@@ -1,31 +1,30 @@
 /* Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. */
 
-import { CustomError } from '../errors/custom.error';
+import { APIError } from '../errors/custom.error';
 
 /**
- * Custom error type to handle API specific errors.
- */
-export class APIError extends CustomError {
-    /**
-     * @constructor
-     *
-     * @param message Error message to be propagated.
-     * @param state OAuth state if available.
-    */
-    constructor(message: string, public innerError?: Error) {
-        super('APIError', message, innerError);
-    }
-}
-
-/**
- * Helper exposing useful Utilities for Excel-Addins.
+ * Helper exposing useful Utilities for Excel Add-ins.
  */
 export class ExcelUtilities {
     /**
-     * Utility to delete a worksheet if it already exists.
-     * @returns true if the sheet had existed before being deleted.
+     * Utility to create (or re-create) a worksheet, even if it already exists.
+     * @returns the new worksheet
      */
-    static async forceCreateSheet(workbook: Excel.Workbook, sheetName: string): Promise<Excel.Worksheet> {
+    static async forceCreateSheet(
+        workbook: Excel.Workbook,
+        sheetName: string,
+        options?: {
```

Comment:
why object? If it is to add future parameters, then i'd suggest creating a separate function that add intended behavior. Keeping functions scoped and to the point is important as opposed to making them extremely extendable.

if this is the only parameter, then just have it as an argument.